cogboard-app/src/test/kotlin/com/cognifide/cogboard/config/service/BoardsConfigServiceTest.kt

```diff
@@ -1,88 +1,74 @@
-        val storage = VolumeStorage(ConfigType.BOARDS, boardConfig, BoardsValidator)
-        val contentRepository = ContentRepository("$boardPath/content")
-        val underTest = BoardsConfigService(storage, contentRepository)
+    private val boardPath = BoardsConfigServiceTest::class.java.getResource("/board").path
+    private val boardConfig = "$boardPath/config-on-backend.json"
+
+    private lateinit var underTest: BoardsConfigService
+
+    @BeforeEach
+    fun initService() {
+        underTest = BoardsConfigService(ContentRepository(boardPath), storage = boards(boardConfig))
+    }
+
+    @Test
+    fun `Expect no widget content in saved config`() {
+        // given
+        val newConfigState = File("$boardPath/config-from-frontend.json").readText()
 
         //when
-        underTest.saveBoardsConfig(JsonObject(uiBoardConfigState))
+        underTest.saveBoardsConfig(JsonObject(newConfigState))
 
         //then
         val contentOnServer = JsonObject(File(boardConfig).readText()).getJsonObject("widgets")
                 .getJsonObject("widgetsById")
-                .getJsonObject("widget1")
+                .getJsonObject("newWidget")
                 .getJsonObject("content")
         assertTrue(contentOnServer.isEmpty)
     }
 
     @Test
-    fun `Expect config merged from server board config and widget states`() {
-        //given
-        val boardPath = BoardsConfigServiceTest::class.java.getResource("/board").path
-        val boardConfig = "$boardPath/server-board-config.json"
-        val storage = VolumeStorage(ConfigType.BOARDS, boardConfig, BoardsValidator)
-        val contentRepository = ContentRepository("$boardPath/content")
-        val underTest = BoardsConfigService(storage, contentRepository)
-
+    fun `Expect widget content merged to config`() {
         //when
         val config = underTest.loadBoardsConfig()
 
         //then
         assertTrue(config.getJsonObject("widgets")
                 .getJsonObject("widgetsById")
-                .getJsonObject("serverWidget1")
+                .getJsonObject("widgetThatIsNotEdited")
                 .getJsonObject("content").containsKey("serverTime"))
     }
 
     @Test
-    fun `Expect all widgets returned`(){
-        //given
-        val boardPath = BoardsConfigServiceTest::class.java.getResource("/board").path
-        val boardConfig = "$boardPath/server-board-config.json"
-        val storage = VolumeStorage(ConfigType.BOARDS, boardConfig, BoardsValidator)
-        val underTest = BoardsConfigService(storage)
-
+    fun `Expect all widgets returned`() {
         //when
         val allWidget = underTest.getAllWidgets()
 
         //then
-        assertTrue(allWidget.containsKey("serverWidget1"))
-        assertTrue(allWidget.size() == 1)
+        assertTrue(allWidget.containsKey("newWidget")
+                || allWidget.containsKey("widgetThatWillBeReplaced")) // depends on execution order
```

Comment:
please..no. keep it separated. Doesn't look nice in my opinion

Uncle Bob says:
> A comment is a failure to express yourself in code.  If you fail, then write a comment; but try not to fail.

Reply:
As this was just a double check I will remove this completely.